bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,164 @@
+		echo >&2 '  with -e WORDPRESS_DB_HOST=hostname:port?'
+		exit 1
+	fi
+
+	# if we're linked to MySQL and thus have credentials already, let's use them
+	: ${WORDPRESS_DB_USER:=${MYSQL_ENV_MYSQL_USER:-root}}
+	if [ "$WORDPRESS_DB_USER" = 'root' ]; then
+		: ${WORDPRESS_DB_PASSWORD:=$MYSQL_ENV_MYSQL_ROOT_PASSWORD}
+	fi
+	: ${WORDPRESS_DB_PASSWORD:=$MYSQL_ENV_MYSQL_PASSWORD}
+	: ${WORDPRESS_DB_NAME:=${MYSQL_ENV_MYSQL_DATABASE:-wordpress}}
+
+	if [ -z "$WORDPRESS_DB_PASSWORD" ]; then
+		echo >&2 'error: missing required WORDPRESS_DB_PASSWORD environment variable'
+		echo >&2 '  Did you forget to -e WORDPRESS_DB_PASSWORD=... ?'
+		echo >&2
+		echo >&2 '  (Also of interest might be WORDPRESS_DB_USER and WORDPRESS_DB_NAME.)'
+		exit 1
+	fi
+
+	if ! [ -e index.php -a -e wp-includes/version.php ]; then
+		echo >&2 "WordPress not found in $(pwd) - copying now..."
+		if [ "$(ls -A)" ]; then
+			echo >&2 "WARNING: $(pwd) is not empty - press Ctrl+C now if this is an error!"
+			( set -x; ls -A; sleep 10 )
+		fi
+		tar cf - --one-file-system -C /usr/src/wordpress . | tar xf -
+		echo >&2 "Complete! WordPress has been successfully copied to $(pwd)"
+		if [ ! -e .htaccess ]; then
+			# NOTE: The "Indexes" option is disabled in the php:apache base image
+			cat > .htaccess <<-'EOF'
+				# BEGIN WordPress
+				<IfModule mod_rewrite.c>
+				RewriteEngine On
+				RewriteBase /
+				RewriteRule ^index\.php$ - [L]
+				RewriteCond %{REQUEST_FILENAME} !-f
+				RewriteCond %{REQUEST_FILENAME} !-d
+				RewriteRule . /index.php [L]
+				</IfModule>
+				# END WordPress
+			EOF
+			chown www-data:www-data .htaccess
+		fi
+	fi
+
+	# TODO handle WordPress upgrades magically in the same way, but only if wp-includes/version.php's $wp_version is less than /usr/src/wordpress/wp-includes/version.php's $wp_version
+
+	if [ ! -e wp-config.php ]; then
+		awk '/^\/\*.*stop editing.*\*\/$/ && c == 0 { c = 1; system("cat") } { print }' wp-config-sample.php > wp-config.php <<'EOPHP'
+// If we're behind a proxy server and using HTTPS, we need to alert Wordpress of that fact
+// see also http://codex.wordpress.org/Administration_Over_SSL#Using_a_Reverse_Proxy
+if (isset($_SERVER['HTTP_X_FORWARDED_PROTO']) && $_SERVER['HTTP_X_FORWARDED_PROTO'] === 'https') {
+	$_SERVER['HTTPS'] = 'on';
+}
+
+EOPHP
+		chown www-data:www-data wp-config.php
+	fi
+
+	# see http://stackoverflow.com/a/2705678/433558
+	sed_escape_lhs() {
+		echo "$@" | sed 's/[]\/$*.^|[]/\\&/g'
+	}
+	sed_escape_rhs() {
+		echo "$@" | sed 's/[\/&]/\\&/g'
+	}
+	php_escape() {
+		php -r 'var_export((string) $argv[1]);' "$1"
+	}
+	set_config() {
+		key="$1"
+		value="$2"
+		regex="(['\"])$(sed_escape_lhs "$key")\2\s*,"
+		if [ "${key:0:1}" = '$' ]; then
+			regex="^(\s*)$(sed_escape_lhs "$key")\s*="
+		fi
+		sed -ri "s/($regex\s*)(['\"]).*\3/\1$(sed_escape_rhs "$(php_escape "$value")")/" wp-config.php
+	}
+
+	set_config 'DB_HOST' "$WORDPRESS_DB_HOST"
```

Comment:
Work on the `wp-config.php` template started in https://github.com/misterbisson/triton-wordpress/pull/1/files#diff-7f5cd2727f11774d6d0dce769cedef7f